live-examples/js-examples/string/string-substring.html

```diff
@@ -0,0 +1,10 @@
+<pre>
+<code id="static-js">var str = 'Mozilla';
+
+console.log(str.substring(1, 2));
+// expected output "oz"
+
+console.log(str.substring(2));
+// expected output "zilla"
+</code>
+</pre>
```

Comment:
This line still seems to end without a newline.

Reply:
There should be one now.